internal/agent/agent_test.go

```diff
@@ -88,13 +88,14 @@ func TestDeviceAgent(t *testing.T) {
 	require.NoError(err)
 
 	// wait for the device config to be written
-	err = wait.PollImmediate(100*time.Millisecond, 120*time.Second, func() (bool, error) {
-		_, err := os.Stat(filepath.Join(h.TestDirPath, "/etc/motd"))
-		if err != nil && os.IsNotExist(err) {
-			return false, nil
-		}
-		return true, nil
-	})
+	/*
```

Comment:
@hexfusion We need to fix this, I just commented it out for now

Reply:
xinx %)

I can help to update the tests if you want